### PR TITLE
docs: Update faq.mdx to add Visual STudio Code Issue and Solution

### PR DESCRIPTION
**Pull request by evo-ksalter**
docs: Added a description on an issue with Visual Studio Code where ASCII escape codes are displayed after switching to Oh My Posh, and a solution.

### Prerequisites

- [X] I have read and understood the [contributing guide][CONTRIBUTING.md].
- [X] The commit message follows the [conventional commits][cc] guidelines.
- [X] Tests for the changes have been added (for bug fixes / features).
- [X] Docs have been added/updated (for bug fixes / features).

### Description

[CONTRIBUTING.md]: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/CONTRIBUTING.md
[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

**Comment by JanDeDobbeleer** [comment]
@evo-ksalter where do you get this fix from? Because the issue originates from their terminal integrations features which needs to be disabled using:

```json
{
  "terminal.integrated.shellIntegration.enabled": false
}
```

**Comment by evo-ksalter** [comment]
@JanDeDobbeleer I linked the source in my update.  Here it is:

https://stackoverflow.com/questions/78468557/unexpected-ansi-escape-codes-on-vs-code-integrated-terminal

File: website/docs/faq.mdx [hunk 1]
```diff
@@ -297,6 +297,27 @@ updated the icons themselves. A backup of the current config can be found in the
...
+ASCII escape sequences like this:
+
+```powershell
+\x1b7\x1b[38\x3b2\x3b89\x3b201\x3b165m \x1b[0m\x1b[38\x3b2\x3b224\x3b222\x3b244min \x1b[38\x3b2\x3b75\x3b149\x3b233m\x1b[1mpwsh\x1b[22m\x1b[27m\x1b[38\x3b2\x3b224\x3b222\x3b244m \x1b[0m\x1b[38\x3b2\x3b224\x3b222\x3b244mat \x1b[38\x3b2\x3b75\x3b149\x3b233m\x1b[1m19:24:06\x1b[22m\x1b[0m\x1b8
+```
+
+The solution is to add the following to the PowerShell configuration in Visual Studio Code settings.json:
+
+```json
+"args": [
```
Comment (JanDeDobbeleer) [inline website/docs/faq.mdx:312]: the actual solution is to disable shell integrations and restart vscode:

```json
{
  "terminal.integrated.shellIntegration.enabled": false
}
```

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

**Comment by JanDeDobbeleer** [comment]
Fixed in the latest release ofvscode

**Pull request closed without merging**